### PR TITLE
Consolidate many tests into a generic "table test"

### PR DESCRIPTION
**Pull request by n1tranquilla**
Closes: #822

### CODE REVIEW

**Review by robinbb — APPROVED** [review]
Wow! A lot of deleted files. Hard to believe that this is essentially (merely) a refactoring.

**Comment by n1tranquilla** [comment]
I went through each carefully to ensure the case was covered in the new file.

File: cypress/e2e/blocks/analytics.cy.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import { GENESIS_BLOCK_BLOCK_HASH } from "../constants";
 
 suite(["@tier1"], "block analytic tab", () => {
-  it("contains the correct elements", () => {
-    cy.visit(`/blocks/${GENESIS_BLOCK_BLOCK_HASH}/analytics`);
-
-    cy.get(".analytics-sm").should("have.lengthOf", 4);
-    cy.get(".analytics-lg").should("have.lengthOf", 2);
-  });
+  // TODO: too expensive to run this query for last 1000 blocks. Re-enable when not
```
Comment (n1tranquilla) [inline cypress/e2e/blocks/analytics.cy.js:4]: This is not an important test. I would not revive the issue.

File: cypress/e2e/summary/overview.cy.js [hunk 2]
```diff
@@ -4,10 +4,10 @@ suite(["@tier1"], "blockchain overview", () => {
     cy.wait(1000);
 
     let summaryItems = [
-      {
-        id: "#uniqueBlockProducers",
-        label: "Unique Producers of last 1000 Blocks",
-      },
+      // {
```
Comment (n1tranquilla) [inline cypress/e2e/summary/overview.cy.js:7]: Not an important test for ME milestone

**Pull request merged**